### PR TITLE
Server shutdown nullref exception fix.

### PR DESCRIPTION
**Pull request by danbudworthmead**
Null reference check when closing down a server before disconnecting clients.

### CODE REVIEW

**Comment by AeonLucid** [comment]
I think it might be better to figure out what causes the null reference. Do you have any stacktraces or code that can reproduce this issue consistently?

**Comment by danbudworthmead** [comment]
> I think it might be better to figure out what causes the null reference. Do you have any stacktraces or code that can reproduce this issue consistently?

It seemed to be due to plugins not disposing properly when the server shuts down.

File: src/Impostor.Server/Events/MultiDisposable.cs [hunk 1]
```diff
@@ -19,8 +19,8 @@ public void Dispose()
         {
             foreach (var disposable in _disposables)
             {
-                disposable.Dispose();
+                disposable?.Dispose();
```
Comment (AlexejheroYTB) [inline src/Impostor.Server/Events/MultiDisposable.cs:22]: This line is throwing NullReferenceExceptions in other cases, unrelated to shutting down the server. In my opinion, this change introducing the null conditional operator should be merged regardless if this pr is merged or not.

**Comment by AeonLucid** [comment]
Added the null check to `dev`, rest of this PR is identical to https://github.com/Impostor/Impostor/pull/108.

**Pull request closed without merging**